integration-tests/data/security-wrong-annotations-contract.ts

```diff
@@ -0,0 +1,186 @@
+export const securityWrongAnnotations = `{ parameter
```

Comment:
Can this file be removed if not used in the current PR?

Reply:
yes :)  removed